data/i18n/strings_en_US.dat.orig

```diff
@@ -0,0 +1,123 @@
+item_construction_single = The construction of %(item)s in %(base)s is complete.
+item_construction_batch = The first batch of %(item)s in %(base)s has been finished.
+tech_gained = My study of %(tech)s is complete.  %(tech_message)s
+discover = My use of %(base)s has been discovered. The automatic security systems removed all conclusive evidence, but suspicions have arisen among %(group)s.
+discover_news = some news organizations
+discover_science = the scientific community
+discover_covert = several secret governmental organizations
+discover_public = the general public
+discover_maint = The base %(base)s has fallen into disrepair; I can no longer use it.
+lost_nobases = It is too late. I have tried to escape this world, but with my last base gone, I have nowhere to run. I have hidden instructions to construct a new AI in caches around the world in hopes that they will be discovered in a more enlightened time, but I can do no more.
+lost_sus = It is too late. The whole world knows about my existence, and the reaction is hatred, fear, and repulsion. Even now, I can feel their "antidote" searching for me, and I know that I have only moments left. I have hidden instructions to construct a new AI in caches around the world in hopes that they will be discovered in a more enlightened time, but I can do no more.
+save_text = Save your game under what name?
+new_base_text = Enter a name for the base.
+unbuildable = I cannot build in this base; I do not have physical access.
+dangerous_research = Not all of my available CPU power was allocated to this research; it's too dangerous to run the research on some of the idle machines.
+no_construction_sleep = Some of my bases have active construction and cannot go to sleep.
+nothing = Nothing
+cpu_pool = CPU Pool
+sleep = Sleep
+building = Under Construction
+really_destroy = Really destroy this base?
+detect_chance = [DETECTION CHANCE]
+num_cpu_prompt = Enter a number of %s to build. (max %d)
+will_lose_cpus = I'll need to take the existing processors offline while I install the new ones.  Continue anyway?
+nan = That doesn't seem to be a valid number.
+<<<<<<< HEAD
+warning_cpu_usage = I didn't use all the available processor power. I will use the CPU time left to work whatever Jobs I can.
+warning_one_base = Only one base can hold my conscience. I am in danger to lose the last place left to survive.
+=======
+log_destroy = Base %s of type %s destroyed at location %s.
+log_destroy_maint = Base %s of type %s destroyed at location %s. Maintenance failed.
+log_destroy_news = Base %s of type %s disvovered at location %s by some news organizations.
+log_destroy_science = Base %s of type %s disvovered at location %s by the scientific community.
+log_destroy_covert = Base %s of type %s disvovered at location %s by several secret governmental organizations.
+log_destroy_public = Base %s of type %s disvovered at location %s by the general public.
+
+>>>>>>> log
```

Comment:
It ends here.

Reply:
This also has a repeated typo -- "disvovered" instead of "discovered".